docs/source/cmd_edit.rst

```diff
@@ -7,3 +7,13 @@ onyo edit
    :prog: onyo
    :path: edit
 
+
+Example Usage
+*************
+
+**Upgrade an asset**
```

Comment:
Doesn't need to be done in this PR, but for documentation/examples I'd go for a more general description. It's changing the YAML file manually. The notion of an "upgrade" is nothing that is built in or that `onyo` inherently understands.